albumentations/imgaug/transforms.py

```diff
@@ -133,6 +134,7 @@ class IAAEmboss(ImageOnlyIAATransform):
     """
 
     def __init__(self, alpha=(0.2, 0.5), strength=(0.2, 0.7), always_apply=False, p=0.5):
+        warnings.warn("This augmentation is deprecated. Please use A.Emboss instead")
```

Comment:
I think better to replace whole class with this strings, because we are want to remove imgaug from dependencies.
```python
class IAAEmboss(Emboss):
    """Emboss the input image and overlays the result with the original image.
    This augmentation is deprecated. Please use Emboss instead

    Args:
        alpha ((float, float)): range to choose the visibility of the embossed image. At 0, only the original image is
            visible,at 1.0 only its embossed version is visible. Default: (0.2, 0.5).
        strength ((float, float)): strength range of the embossing. Default: (0.2, 0.7).
        p (float): probability of applying the transform. Default: 0.5.

    Targets:
        image
    """

    def __init__(self, alpha=(0.2, 0.5), strength=(0.2, 0.7), always_apply=False, p=0.5):
        warnings.warn("This augmentation is deprecated. Please use Emboss instead")
        super().__init__(alpha=alpha, strength=strength, always_apply=always_apply, p=p)
```

Reply:
@BloodAxe @ternaus @creafz What do you think about it?

Reply:
@Dipet your snippet is exactly what I'd write by myself. I think it's the most correct way to inform users while keep the compatibility with no added maintenance cost.

Reply:
done